### PR TITLE
Eslint

### LINKED ISSUE #4: Add ESLint + Prettier

### PR DESCRIPTION
**Pull request by Beker94**
added Eslint+ Prettier

Issue #4

### CODE REVIEW

**Comment by enheit** [comment]
Please, make pull request into the `develop` branch instead of `master`.

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: "all",
+    singleQuote: true,
+    printWidth: 120,
+    tabWidth: 4
```
Comment (enheit) [inline .prettierrc.js]: I would recommend to use `tabWith: 2`. 

What kind of indent is used here `tab` or `space`? If `tab` than it should be changed to `space`.

**Review by ivan-bonk — APPROVED** [review]
Good job!

**Review by Pashkaua — APPROVED** [review]
Look nice !

**Review by okhabr — APPROVED** [review]
Seems ok.

**Review by enheit — APPROVED** [review]

**Pull request merged**